v2g-liberty/rootfs/root/homeassistant/packages/v2g_liberty/v2g_liberty_package.yaml

```diff
@@ -242,8 +242,14 @@ input_number:
     step: 1
     icon: mdi:timer-lock-outline
     mode: box
-    unit_of_measurement: "hrs"
-
+    unit_of_measurement: hrs
+
+template:
+  - binary_sensor:
+      - name: is_charge_power_different_from_schedule
+        device_class: problem
+        state: >
+          {{ ((states('sensor.current_scheduled_charging_power')|int - states('sensor.charger_real_charging_power')|int)|abs) > 500 }}
```

Comment:
This does not have to be a problem perse, does it?
It could also mean that the load balancer is doing a good job.

Reply:
True, it would not even have to be a problem if it lasts for a longer period (the intended use is to check this). On the other hand, the (only) alternative would be "generic", other device classes don't match at all. So device_class: problem is the best match. Maybe the threshold should be a user setting? 
Do you see a problem in using device_class: problem?